docs/website/versioned_docs/version-3.0.0/user-guides/quickstart.md

```diff
@@ -389,6 +492,204 @@ Your Devfile has been successfully deployed
 
 Your application has now been deployed to the Kubernetes cluster with Deployment, Service, and Ingress resources.
 
+Test your application by visiting the `DOMAIN_NAME` variable that you had set in the `devfile.yaml`.
```

Comment:
I wonder why this statement doesn't show up on the [quickstart guide](https://odo.dev/docs/3.0.0/user-guides/quickstart) on the site. 🤔 

It shows up only once at the end of the guide. Maybe I'm missing something.

Reply:
Yup! Since this is part of `odo deploy`, you can visit the ingress you made with the domain name at the end of the quickstart guide.